### DISCUSSION TITLE
Ability to download shows

### DISCUSSION THREAD

**Discussion by senorborkbork101** [f1multiviewer/issue-tracker#126] [Feature requests] [answered]
Hi,

One of the big attractions to me with RaceControl was the ability to download and archive old races. Is this something that will be coming to MV?

I've only discovered MV today and so far I'm really impressed. Great work by all involved!

**Comment by f1multiviewer** [comment, marked as answer]
It will not, but RaceControl and f1viewer both still allow this. F1MV won't add this, since it's focussing on providing a better viewing experience.
